src/solver/conv_asm_implicit_gemm_wrw_gtc_dynamic_xdlops.cpp

```diff
@@ -516,11 +844,20 @@ ConvAsmImplicitGemmGTCDynamicWrwXdlops::GetSolution(const ConvolutionContext& ct
 
     kernel_name = kernel_configs[kernel_index].GetKernelName();
 
-    // std::cout << "tuple=" << grid_size << " " << log2_gemm_k_global_splits << std::endl;
+    // MIOPEN_LOG_I2(kernel_name << " with groups for reduction: "
+    //                           << (1 << log2_gemm_k_global_splits));
 
-    result.workspce_sz = 0;
+    const auto workspce_sz = GetWorkspaceSize(ctx);
```

Comment:
Please rename `workspce_sz` to `required_workspace_size` or something like that.

Reply:
renamed.